Android/ZensoftHRCRM/app/src/main/java/com/erkprog/zensofthrcrm/ui/BaseFragment.java

```diff
@@ -0,0 +1,30 @@
+package com.erkprog.zensofthrcrm.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+
+public abstract class BaseFragment extends Fragment {
+
+  /**
+   * This method changes action bar title.
+   * Set null if you don't want any changes.
+   */
+
+  protected abstract String setTitle();
```

Comment:
Incorrect method name. This should be getTitle. 
Or change setTitle method to void and implement changing toolbar name in active